src/develop/components/SearchDoc.js

```diff
@@ -143,6 +162,16 @@ var SearchDoc = React.createClass({
             this.state.value + "\" suggestions={" +
             stringify(this.state.simpleSuggestions) + "}/>"}</code></pre>
 
+          <h3>Inline, Value, Rich Suggestions (onDOMChange and onSelect)</h3>
+          <div className="example">
+            <Search inline={true} value={this.state.value}
+              suggestions={this.state.richSuggestions}
+              onDOMChange={this._onDOMChange} onSelect={this._onSelect} />
```

Comment:
I think we should update all examples in this file to use onDOMChange and onSelect, since onChange is deprecated.